tests/test_dimming_actuator.py

```diff
@@ -0,0 +1,138 @@
+"""Test class to test the DimmingActuator device."""
+
+from unittest.mock import AsyncMock
+
+import pytest
+
+from abbfreeathome.api import FreeAtHomeApi
+from abbfreeathome.devices.dimming_actuator import DimmingActuator
```

Comment:
Reference module files directly (vs installed module)

```suggestion
from src.abbfreeathome.api import FreeAtHomeApi
from src.abbfreeathome.devices.dimming_actuator import DimmingActuator
```